source/Various/File1.lagda

```diff
@@ -0,0 +1,55 @@
+Alice Laroche , 26th September 2023
+\begin{code}
+
+{-# OPTIONS --safe --without-K --exact-split #-}
+
+open import MLTT.Spartan
+open import UF.Univalence
+
+module Various.File1
+        (ua : Univalence)
+        (𝓤 : Universe)
+       where
+
+open import UF.Base
+open import UF.DiscreteAndSeparated
+open import UF.Embeddings
+open import UF.Subsingletons
+open import Iterative.Multisets 𝓤
+open import Iterative.Multisets-Addendum ua 𝓤
+open import Iterative.Sets ua 𝓤
+open import Iterative.Sets-Addendum ua 𝓤
+open import Iterative.Ordinals ua 𝓤
+
+𝟘ⱽ-is-transitive-iset : is-transitive-iset 𝟘ⱽ
+𝟘ⱽ-is-transitive-iset v₁ v₂ ()
```

Comment:
We would prefer to use `𝟘-elim`, although we are that in some places in TypeTopology absurd patterns are still used.